tests/integration/test_integration.py

```diff
@@ -108,7 +71,7 @@ async def test_given_correct_config_when_deploy_and_scale_then_status_is_active(
 
         await ops_test.model.wait_for_idle(apps=[APPLICATION_NAME], status="active", timeout=1000)
 
-    async def test_given_tls_requirer_is_deployed_and_related_then_certificate_is_created_and_passed_correctly(  # noqa: E501
+    async def test_given_tls_requirer_is_deployed_and_related_when_provide_certificate_then_certificate_is_passed_correctly(  # noqa: E501
```

Comment:
Could we also test the situation that action inputs are not correct ?

Reply:
Unit tests validate this, if an invalid input is given the action will fail.
I'm not convinced it would bring any value here as the action will fail locally and not affect the state of neither this or the remote charm.
If you have any suggestions please let me know :)

Reply:
OK, this is enough.